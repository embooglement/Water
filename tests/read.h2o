print("Type in your name: ");
let x = readln();
print("\nYour name is:", x);
print("\nAlso", x, read(), read());
