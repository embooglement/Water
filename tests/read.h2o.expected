Type in your name: 
Your name is: Kevin
Also Kevin is awesome
